doc/source/rllib/doc_code/new_api_stack.py

```diff
@@ -14,14 +15,16 @@
         enable_rl_module_and_learner=True,
         enable_env_runner_and_connector_v2=True,
     )
+    .resources(
+        num_cpus_for_main_process=1,
+    )
     # We are using a simple 1-CPU setup here for learning. However, as the new stack
     # supports arbitrary scaling on the learner axis, feel free to set
-    # `num_learner_workers` to the number of available GPUs for multi-GPU training (and
-    # `num_gpus_per_learner_worker=1`).
-    .resources(
-        num_learner_workers=0,  # <- in most cases, set this value to the number of GPUs
-        num_gpus_per_learner_worker=0,  # <- set this to 1, if you have at least 1 GPU
-        num_cpus_for_local_worker=1,
+    # `num_learners` to the number of available GPUs for multi-GPU training (and
+    # `num_gpus_per_learner=1`).
```

Comment:
It's not a necessary limit and we won't error out if the user sets it to >1. It could be for example that they have a model that doesn't fit on one GPU. We should leave this option flexible for future pipelining/model-parallelism support, I think.